README.md

```diff
@@ -2,13 +2,25 @@
 
 Common queries for Canvas data
 
-## Current Term Courses Info
-1. We fetch current term courses information, student assignment, grades and background information from UDP. All the SQL quries associated from that are in `current_term_courses`
-2. Athletic dept is the main user of this view, which are Materialized views.
-3. MatViews are created on the BigQuery Context store and refreshed 1PM EST
+## Creating Scheduled Queries
+1. Go to BiqQuery Console for [scheduled queries](https://console.cloud.google.com/bigquery/scheduled-queries?project=udp-umich-prod) and click on `+ Create scheduled query in editor`
+2. Enter your query in the Editor and click `scheduled` button
+3. Add following in the pop-up window
+   1. Details and schedule: `datamart_name__table_name_you_want_to_give`. For example: `athletic_department_views__v1_student_current_term_course`
+   2. Schedule option: `days` and at `18:00`. Don't need to do anything for `Start now` and `End never` elements.
+   3. Destination for Query result: 
+      1. Check the checkbox `Set a destination table for query results`
+      2. Choose the data set name `athletic_department_views`
+      3. Give a table name for example: `v1_student_current_term_course`
+4. Destination table write preferance: `Overwrite table`
 
-## Auto-Refreshed Materialized Views
-Details are here: https://resources.unizin.org/display/UDP/UDP+Context+store
+## Updating a Scheduled queries
+1. There is no update of scheduled query it's always Delete and create a new table. This limitation is from BQ side
```

Comment:
Adding a break/dash character into the sentence:

There is no update of scheduled query - it's always Delete and create a new table

Reply:
Done